pressbooks/pressbooks-aldine
# fix: add additional selector to contact class

Fix for #456

To test:
1. Create a page called 'Contact' with the permalink '/contact`
2. Visit the page and observe the broken styling (all content forced width of 800 px, messed up background, padding, etc.)
3. Check out this branch
4. Visit the page again and observe styling looks as expected for regular Aldine pages
